include/drivers/i2c/slave/eeprom.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file
+ *
+ * @brief Public APIs for the I2C EEPROM Slave driver.
+ */
+
+/*
+ * Copyright (c) 2017 BayLibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef I2C_EEPROM_SLAVE_H
+#define I2C_EEPROM_SLAVE_H
+
```

Comment:
There should be a doxygen defgroup to contain these APIs (and for them to be displayed in the generated API docs)

Reply:
Done